oak_functions/README.md

```diff
@@ -50,89 +50,7 @@ from the conventional computing model.
 <!-- https://docs.google.com/drawings/d/1ZPeJ93IkyOOJVI8CFSbEeEKn6wVozB-d6E1SekK2QyQ/edit -->
 <img src="images/ComputingModel.png" width="1000">
 
-## Features
```

Comment:
Perhaps add a paragraph here with a link to the loader directory to clarify how this is actually used?

Reply:
Good point, done, thank you!